test/use_case/search_nearby/SearchNearbyInteractorTest.java

```diff
@@ -0,0 +1,91 @@
+package use_case.search_nearby;
+
+import data_access.InMemoryEventsDataAccessObject;
+import entity.Events.CommonEvent;
+import entity.Events.Event;
+import entity.Location.CommonLocationFactory;
+import entity.Location.Location;
+import org.junit.jupiter.api.Test;
+import use_case.search_event.SearchEventInputData;
+import use_case.search_event.SearchEventOutputBoundary;
+import use_case.search_event.SearchEventOutputData;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SearchNearbyInteractorTest {
+    @Test
+    void searchSuccess() {
+        CommonLocationFactory factory = new CommonLocationFactory();
+        try {
```

Comment:
Instead of repeatedly making each object for each test, I suggest that you create an init() method that initializes all common objects that will be used for all tests